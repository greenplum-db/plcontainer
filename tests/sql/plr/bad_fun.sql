-- should error out but should not crash PG
CREATE OR REPLACE
FUNCTION r_bad_fun()
RETURNS int4 AS
$$
# container: plc_r_shared
  deadbeef <- function(,bad) {}
  42
$$
LANGUAGE plcontainer;
select r_bad_fun();
