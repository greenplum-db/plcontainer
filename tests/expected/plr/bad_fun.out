-- should error out but should not crash PG
CREATE OR REPLACE
FUNCTION r_bad_fun()
RETURNS int4 AS
$$
# container: plc_r_shared
  deadbeef <- function(,bad) {}
  42
$$
LANGUAGE plcontainer;
ERROR:  R interpreter parse error
DETAIL:  R parse error caught in "
  deadbeef <- function(,bad) {}
  42
".
